packages/asc-assets/README.md

```diff
@@ -25,29 +25,43 @@ return <IconName />
 
 ### Fonts
 
-Below an example of how to use with CopyWebpackPlugin:
+> **Note**: Some of the fonts that are used in Amsterdam are licensed, before you include these fonts in your application make sure that you have obtained a licence to do so.
 
-```js
-new CopyWebpackPlugin([
-  {
-    from: './node_modules/@datapunt/asc-assets/static/fonts',
-    to: 'fonts',
-  },
-])
-```
+There are currently two ways of including the fonts used in your application, you can either copy them in from this package in your build or include a version from the Amsterdam CDN. Note that if you want to use the version hosted by Amsterdam you must have an application that runs in the `amsterdam.nl` domain space.
 
-don't forget to include the css!
+#### Using the Amsterdam CDN
 
-```scss
-@import '~@datapunt/asc-assets/static/fonts/fonts.css';
+Add the following code to your `index.html` file:
+
+```html
+<!-- This script is needed as specified by the font license -->
+<script src="https://static.amsterdam.nl/fonts/mtiFontTrackingCode.min.js"></script>
+<link href="https://static.amsterdam.nl/fonts/fonts.css" rel="stylesheet" />
 ```
 
-## Optimized SVGs
+There is nothing more to do, the fonts should not be available in your application.
```

Comment:
There is nothing more to do, the fonts should `not` -> `now` be available in your application.

Reply:
Thanks for checking, fixed the typos.